urlshort/README.md

```diff
@@ -0,0 +1,36 @@
+# URL Shortener
+Exercise details from https://gophercises.com/exercises/urlshort
```

Comment:
How are you able to know that your code is correct without tests??

Reply:
The exercise was just to implement handler.go as main.go was already provided